Sources/MapboxMaps/Ornaments/Compass/MapboxCompassOrnamentView.swift

```diff
@@ -33,15 +34,15 @@ internal class MapboxCompassOrnamentView: UIButton {
         didSet {
             let adjustedBearing = currentBearing.truncatingRemainder(dividingBy: 360)
             updateVisibility()
-            self.transform = CGAffineTransform(rotationAngle: -adjustedBearing.toRadians())
+            self.containerView.transform = CGAffineTransform(rotationAngle: -adjustedBearing.toRadians())
         }
     }
 
     required internal init(visibility: OrnamentVisibility) {
         super.init(frame: .zero)
         translatesAutoresizingMaskIntoConstraints = false
         self.visibility = visibility
-        alpha = visibility == .visible ? 1 : 0
+        containerView.alpha = visibility == .visible ? 1 : 0
```

Comment:
if `containerView` is a `UIView` we should use the `isHidden` property. Messing with alphas for visibility is not right